tests/test_auth.py

```
@@ -0,0 +1,543 @@
+                "github_token": "ghp_test123",
+                "jira_email": "a@b.com",
+                "jira_token": "jira_tok",
+            },
+            cookies={"jji_username": "tokenuser"},
+        )
+        assert resp.status_code == 200
+        # Get tokens back
+        resp = client.get("/api/user/tokens", cookies={"jji_username": "tokenuser"})
+        assert resp.status_code == 200
+        data = resp.json()
+        assert data["github_token"] == "ghp_test123"
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "github_token"

(S105)